spec/src/main/asciidoc/ch05-metamodel-api.adoc

```diff
@@ -72,13 +72,14 @@ metamodel class must contain a field declaration as follows:
 +
 [source,java]
 ----
-public static final String Y = "y";
+public static final String Y_ = "y";
```

Comment:
remove underscore, add disclaimer about non-portable behaviour in case of collisions